fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -315,6 +315,7 @@ public class SessionVariable implements Serializable, Writable, Cloneable {
     public static final String PARSE_TOKENS_LIMIT = "parse_tokens_limit";
 
     public static final String ENABLE_SORT_AGGREGATE = "enable_sort_aggregate";
+    public static final String ENABLE_PARALLEL_MERGE = "enable_parallel_merge";
```

Comment:
enable_parallel_sort_merge ?

Reply:
It's a temporeary session variable, I'll delete it in the final pr.